bundler/lib/dependabot/bundler/update_checker/shared_bundler_helpers.rb

```diff
@@ -31,8 +31,7 @@ module SharedBundlerHelpers
         PATH_REGEX = /The path `(?<path>.*)` does not exist/
 
         module BundlerErrorPatterns
-          # The `set --global` optional part can be made required when Bundler 1 support is dropped
-          MISSING_AUTH_REGEX = /bundle config (?:set --global )?(?<source>.*) username:password/
+          MISSING_AUTH_REGEX = /bundle config set --global (?<source>.*) username:password/
 
```

Comment:
As mentioned here, after dropping bundler 1 support regex is simplified to work on bundler v2 as mentioned in the shared link. 

- https://github.com/dependabot/dependabot-core/issues/6457#issuecomment-1673532403